content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -463,8 +461,8 @@ a special whole-number notation known in Kubernetes as a *quantity*.  For exampl
 the quantity `10500m` would be written as `10.5` in decimal notation.  The metrics APIs
 will return whole numbers without a suffix when possible, and will generally return
 quantities in milli-units otherwise.  This means you might see your metric value fluctuate
-between `1` and `1500m`, or `1` and `1.5` when written in decimal notation.  See the
-[glossary entry on quantities](/docs/reference/glossary?core-object=true#term-quantity) for more information.
+between `1` and `1500m`, or `1` and `1.5` when written in decimal notation. See the
+{{< glossary_tooltip term_id="quantity" >}} for more information.
```

Comment:
I'd drop the sentence containing the hyperlink and make
```markdown
a special whole-number notation known in Kubernetes as a *quantity*.  For example,
```
be
```markdown
a special whole-number notation known in Kubernetes as a
{{< glossary_tooltip term_id="quantity" text="quantity" >}}.  For example,
```

Reply:
Agreed. This is much cleaner.